contentcuration/contentcuration/settings.py

```diff
@@ -204,12 +204,12 @@
 WSGI_APPLICATION = 'contentcuration.wsgi.application'
 ASGI_APPLICATION = 'contentcuration.asgi.application'
 
-
+CHANNELS_DB = 2
 CHANNEL_LAYERS = {
     'default': {
         'BACKEND': 'channels_redis.core.RedisChannelLayer',
```

Comment:
@ozer550 It would be nice to update this to the `RedisPubSubChannelLayer`! It should be more efficient